spring-jdbc/src/main/java/io/crate/spring/jdbc/samples/da/ImagesJDBCTemplateDao.java

```diff
@@ -0,0 +1,134 @@
+package io.crate.spring.jdbc.samples.da;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.http.HttpStatus;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.lang.NonNull;
+import org.springframework.stereotype.Component;
+
+import io.crate.spring.jdbc.samples.domain.Image;
+
+@Component
+public class ImagesJDBCTemplateDao implements ImagesDao {
+
+    private static final String IMAGE_TABLE = "guestbook_images";
+
+    @Value("${cratesample.cratedb.blobEndpointURL:http://localhost:4200/_blobs/}")
+    private String blobEndpointURL;
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    private HttpClient client = HttpClient.newHttpClient();
+
+    @Override
+    public boolean imageExists(@NonNull final String digest) {
+
+        boolean result = false;
+
+        var request = HttpRequest.newBuilder(this.createBlobUri(digest))
+                .method("HEAD", HttpRequest.BodyPublishers.noBody()).build();
+
+        try {
+            HttpResponse<Void> response = client.send(request, HttpResponse.BodyHandlers.discarding());
+            result = response.statusCode() == HttpStatus.OK.value();
+        } catch (IOException | InterruptedException e) {
+            throw new DataIntegrityViolationException("Failed to call blob endpoint", e);
+        }
+
+        return result;
+    }
+
+    @Override
+    public List<Image> getImages() {
+        return this.jdbcTemplate.query(String.format("SELECT digest, last_modified " + "FROM %s " + "ORDER BY 2 DESC",
```

Comment:
Same here, `String.format` seems unnecessary.

Reply:
Same again. Same way as original JDBC sample

Reply:
And btw. I think it's a good idea to have the table names not part of the statement. If you want to change the table names, you'll have to change tens of statements correctly!